app/main.py

```diff
@@ -1,3 +1,71 @@
+import operator
+from typing import Union, Callable
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: float) -> None:
+        self.km = km
```

Comment:
What if user passes wrong data type to our class? Should we have some validation in place?

Reply:
I see you haven't started topic with properties and descriptors yet. So if you can't do it right now you can ignore this comment. But if you can do it you will be amazing :)

Reply:
Nice! Now on object creation we will reject incorrect values and our program will be more stable! Later we will learn how to handle it in more pythonic way